go/apps/dialogue/tests/dummy_polls.js

```diff
@@ -65,6 +67,24 @@ var simple_poll = {
         entry_endpoint: {uuid: "endpoint-10"},
         // end specific
         text: "Thank you for taking our survey"
+    },
+    {
+        uuid: "http_json-1",
+        name: "json post 1",
+        type: "httpjson",
+        url: "www.foo.bar", 
+        method: "GET",
+        entry_endpoint: {uuid: "endpoint-12"},
+        exit_endpoint: {uuid: "endpoint-13"}
+    },
+    {
+        uuid: "http_json-2",
+        name: "json post 2",
+        type: "httpjson",
+        url: "www.foo.bar", 
+        method: "POST",
+        entry_endpoint: {uuid: "endpoint-15"},
+        exit_endpoint: {uuid: "endpoint-13"}
```

Comment:
I think for sanity this should be `endpoint-16` and we should add `endpoint-16 -> endpoint-10` to the connection list since endpoint IDs are supposed to be unique.